src/app/containers/create-movie/formField.jsx

```diff
@@ -0,0 +1,23 @@
+import { Field } from "formik"
+
+export const FormField = ({ label, placeholder, name, type, error }) => {
+    const nameParts = name.split('.');
+  
+    let transformedName =  nameParts[1] != undefined?`${nameParts[0]}[${nameParts[1]}].${nameParts[2]}`:`${nameParts[0]}`;
+    if(nameParts[0] == "crew"){
```

Comment:
Instead of edge casing a component that can be reusable to handle "crew", we could leave the FormField as a template and create a CrewFormField that would handle the transformed name for the crew. Also, why do we need such a long name for the input name?